api/database/migrations/2021_09_17_135548_create_candidate_search_requests_table.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateCandidateSearchRequestsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('candidate_search_requests', function (Blueprint $table) {
+            $table->id();
```

Comment:
Should this be a uuid type to match the other models use of uuid instead of an id?